app/config/permissions.neon

```diff
@@ -306,6 +306,23 @@ permissions:
       conditions:
           - exercise.isAuthor
 
+    - allow: true # Admin of the group of an exercise has most of the rights to the exercise
```

Comment:
Wouldn't it be better to add condition `exercise.isSuperGroupAdmin` to above `exercise.isAuthor` and separate `delete` actions into new permission rule?

Reply:
Conditions are joined with `&&`, so this wouldn't work. I still welcome suggestions for a good `||` syntax, though.

Reply:
I though something like `exercise.isAuthorOrSuperGroupAdmin` method... but not sure if it is a good idea

Reply:
That sounds way too specific for my taste.

Reply:
Ok, fair enough...